src/flow-spec/IBlock.ts

```diff
@@ -289,14 +317,12 @@ export function wrapInExprSyntaxWhenAbsent(expr: string): string {
 /**
  * Set a property on the contact contained in the flow context.
  */
-export function setContactProperty<BLOCK_CONFIG extends ISetContactPropertyBlockConfig>(
+export function setContactProperty<BLOCK_CONFIG extends IBlockConfig | ISetContactPropertyBlockConfig>(
```

Comment:
Why doesn't ISetContactPropertyBlockConfig extend IBlockConfig? That seems like a solution that won't scale in the code, and requires us to write more code anytime we want to mention a type of IBlockConfig or have a block that can edit contact properties (and so many of them do/will)

Reply:
fixed with the above task
> improve ISetContactPropertyBlock interfaces declaration